src/tracks/InstrumentTrack.cpp

```diff
@@ -304,6 +311,25 @@ void InstrumentTrack::processInEvent( const MidiEvent& event, const MidiTime& ti
 				}
 				else
 				{
+					if (isSustainPedalPressed())
```

Comment:
I think you can write this into an `if else` with the `else` above to simplify it further.
And then lines 324-326 can get one tab each.